pkg/goGit/goGit.go

```diff
@@ -14,13 +15,28 @@ import (
 	logr "github.com/sirupsen/logrus"
 )
 
-func UpdateLocalCopies(repos []gitapi.Repository, config *config.Config, user *config.User) {
+func UpdateLocalCopies(repos []gitapi.Repository, config *config.Config, account *config.Account) {
+	i := 0
 	for _, repo := range repos {
+		if i >= 5 {
+			logr.Infof("[Git] Stopping after 5 repositories ...")
+			break
+		}
+		i++
```

Comment:
Debug limitation again. Please remove and check the code you commit more carefully in the future.